bot.py

```diff
@@ -15,18 +15,6 @@
 
 CLIENT = TelegramClient(SESSION, API_ID, API_HASH)
```

Comment:
Is `CLIENT` instance a constant (i.e., doesn't change it's state whatsoever and has 0 side-effects)? If not, it shouldn't be in upper case.

Reply:
The `CLIENT` instance will never change, it configure before start only once. So it is the constant